src/java.desktop/unix/native/libawt/awt/initIDs.c

```diff
@@ -40,13 +40,10 @@
  * which are used in the win32 awt.
  */
 
-JNIEXPORT jfieldID colorValueID;
-
 JNIEXPORT void JNICALL
 Java_java_awt_Color_initIDs
   (JNIEnv *env, jclass clazz)
 {
-    colorValueID = (*env)->GetFieldID(env, clazz, "value", "I");
 }
```

Comment:
Shall we remove `Color.initIDs`? Its implementation is now empty for all platforms.

Reply:
Sure, will do.

Reply:
Found and removed a few more initIDs methods that were empty on all platforms.

Reply:
Sounds reasonable. Should I move the cleanup of Color as well?

Reply:
> Should I move the cleanup of Color as well?

Removing `colorValueID` could be left here, I think: it removes an exported symbol which is no longer used.

In addition to it, it makes `Java_java_awt_Color_initIDs` empty.

Reply:
Ok, I'll revert the initIDs removals, and will file a follow-up PR once this one is merged.

Reply:
> …will file a follow-up PR once this one is merged

You can submit it right away, so that we'll not forget about it.

Reply:
Filed [JDK-8318091](https://bugs.openjdk.org/browse/JDK-8318091) for removal of initIDs; the changes there overlap with this PR, and I'd rather not create dependent PRs.